### PR TITLE
don't create meaningless sensitivities, closes #450

### LINKED ISSUE #450: sens only gives variable sensitivities, but accepts signomial input
Is doing so even possible? I am skeptical.

### PR DESCRIPTION
**Pull request by bqpd**
@whoburg, ready for review

### CODE REVIEW

File: gpkit/solution_array.py [hunk 1]
```diff
@@ -80,17 +80,7 @@ def sens(self, nomial):
```
Comment (whoburg) [inline gpkit/solution_array.py]: should the docstring be updated? What are the valid input types for `nomial`? Variable? Monomial? Signomial?

Comment (bqpd) [inline gpkit/solution_array.py]: Anything that's in the keymap or has a .key attribute.

Comment (whoburg) [inline gpkit/solution_array.py]: (I'm referring to the `sens` method -- looks like this inline comment is showing up in an unexpected place)

Then should `sens(self, nomial)` change to `sens(self, key)` and the docstring be updated to remove the part about signomial input?

**Comment by whoburg** [comment]
@bqpd, this is waiting on your reply to the docstring comment/question.

**Comment by bqpd** [comment]
Responded to code review!

**Comment by bqpd** [comment]
@whoburg please merge this if it passes your review, it's an issue in Next Release

**Pull request merged**